apps/full-site-editing/package.json

```
@@ -131,6 +131,9 @@
 	},
 	"devDependencies": {
 		"@types/wordpress__plugins": "^2.3.6",
-		"@wordpress/eslint-plugin": "^6.0.0"
+		"@babel/preset-typescript": "^7.10.1",
+		"@types/jest": "^26.0.3",
```

Let's match root version:

```suggestion
		"@types/jest": "^25.2.1",
```

https://github.com/Automattic/wp-calypso/blob/9a5f2906cc1da31e230efd2000551a176daeb6e0/package.json#L193